plugin/build.gradle

```diff
@@ -62,6 +62,15 @@ dependencies {
     testImplementation group: 'commons-io', name: 'commons-io', version: '2.15.1'
     implementation group: 'org.apache.commons', name: 'commons-text', version: '1.10.0'
     implementation 'com.jayway.jsonpath:json-path:2.9.0'
+
+    implementation "org.opensearch.client:opensearch-java:2.10.2"
+
+    configurations.all {
+        resolutionStrategy.force 'org.apache.httpcomponents.core5:httpcore5:5.2.4'
+        resolutionStrategy.force 'org.apache.httpcomponents.core5:httpcore5-h2:5.2.4'
+        resolutionStrategy.force 'jakarta.json:jakarta.json-api:2.1.3'
+        resolutionStrategy.force "org.opensearch.client:opensearch-rest-client:${opensearch_version}"
+    }
```

Comment:
I'm not sure you need all of these dependencies, they're specific to the feature branch.

Reply:
Removed